src/Loop.php

```diff
@@ -59,155 +59,162 @@ public static function stop()
     /**
      * Defer the execution of a callback.
      *
-     * @param callable $callback The callback to defer.
+     * @param callable(mixed $data, string $watcherIdentifier) $callback The callback to defer.
```

Comment:
@bwoebi I think @trowski might be right, it might be required to remove it from an array or so.

But it seems like we don't use it in Amp, but we have it. https://github.com/amphp/aerys/blob/2676b2a7bb9d4bcacf82a633b754a5f18e40e26a/lib/Websocket/Rfc6455Endpoint.php#L209-L242